command/meta.go

```diff
@@ -198,6 +207,9 @@ func generalOptionsUsage() string {
   -tls-skip-verify
     Do not verify TLS certificate. This is highly not recommended. Verification
     will also be skipped if NOMAD_SKIP_VERIFY is set.
+
+  -token
+    Provide an ACL token to access privilated information
```

Comment:
The SecretID of an ACL token to use to authenticate API requests with.

Reply:
We should document that `NOMAD_TOKEN` env var will be sourced as well. Usually env vars preferred for sensitive values like token.